lizard_auth_client/conf.py

```diff
@@ -1,10 +1,17 @@
 # -*- coding: utf-8 -*-
+
 from __future__ import unicode_literals
 
 from appconf import AppConf
+from django import VERSION
 from django.conf import settings
-from django.utils.translation import ugettext_lazy as _
 
+if VERSION < (2,):
```

Comment:
Dit zal wel niet nodig zijn? Alleen lizard+3di gebruiken dit nog, dus ik gok dat django 1.8 er al wel uit is?

Reply:
Ach, kan nul kwaad toch?

Reply:
Het kan geen kwaad, maar heeft geen nut. 1.11 was 3 jaar geleden EOL en heeft python 3.7 (waar de tests nu van uit staan) als laatste python versie.